client/src/components/Community/Community.js

```diff
@@ -1,52 +1,47 @@
 import React from 'react';
+import { Link } from "react-router-dom";
 import 'bootstrap/dist/css/bootstrap.min.css';
 import './Community.css';
-import { Carousel, Card } from "react-bootstrap"
+import union from './union.svg';
+import list from './CommunityList';
+import loader from './loader.svg';
 
 function Community() {
   return (
     <div className="community-component">
-      <div className="Society-cards">
-        <h1 className="heading">Societies under Geeks</h1>
-        <Carousel>
-          <Carousel.Item>
-            <Card style={{ width: '18rem' }}>
-              <Card.Body>
-                <Card.Title>Card Title</Card.Title>
-                <Card.Subtitle className="mb-2 text-muted">Card Subtitle</Card.Subtitle>
-                <Card.Text>
-                  Some quick example text to build on the card title and make up the bulk of
-                  the card's content.
-                </Card.Text>
-              </Card.Body>
-            </Card>
-          </Carousel.Item>
-          <Carousel.Item>
-            <Card style={{ width: '18rem' }}>
-              <Card.Body>
-                <Card.Title>Card Title</Card.Title>
-                <Card.Subtitle className="mb-2 text-muted">Card Subtitle</Card.Subtitle>
-                <Card.Text>
-                  Some quick example text to build on the card title and make up the bulk of
-                  the card's content.
-                 </Card.Text>
-              </Card.Body>
-            </Card>
+      <div className="union">
+          <img src={union} className="union-img" alt="union"/>
+      </div>
+      <div className="container-fluid main row justify-content-around">
+
```

Comment:
change this <div> as <div className="container-fluid main> and add another <div> after this as <div className="row justify-content-around

Reply:
Thanks for pointing out! :wink:

Reply:
Done.